distribution/docs/src/main/resources/_contents/_tables/org.codice.admin.ui.configuration-table-contents.adoc

```diff
@@ -0,0 +1,76 @@
+.[[org.codice.admin.ui.configuration]]Admin UI Configuration
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Enable System Usage message
+|systemUsageEnabled
+|Boolean
+|Turns on a system usage message,which is shown when the Admin Application is opened
+|false
+|true
+
+|System Usage Message Title
+|systemUsageTitle
+|String
+|A title for the system usage message when the application is opened
+|
+|true
+
+|System Usage Message
+|systemUsageMessage
+|String
+|A system usage message to be displayed to the user each time the user opens the application
+|
+|true
+
+|Show System Usage Message once per session
+|systemUsageOncePerSession
+|Boolean
+|With this selected,the system usage message will be shown once for each browser session.  Uncheck this to have the usage message appear every time the admin page is opened or refreshed.
```

Comment:
Missing space after comma. Also, looks like two spaces after the period again. Is that our house style?